css/main.css

```
@@ -8,3 +8,26 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+header{
```

usar classe no lugar dessa tag,
tipo assim: 
```suggestion
.header{
```

---

belezaaa !! pode deixarrr